client/src/pages/api/update/index.js

```diff
@@ -0,0 +1,15 @@
+import prisma from "../../../lib/prisma";
+
+export default async function handle(req, res) {
+  const { img, author, title } = req.body;
+  const { id } = req.params;
+  const foodItem = await prisma.food.update({
+    where: { id: Number(id) },
+    data: {
+      img,
+      author,
+      title,
+    },
+  });
+  res.json(foodItem);
+}
```

Comment:
See comment above regarding folder structure and naming conventions.

Reply:
yes